README.md

```diff
@@ -535,34 +535,148 @@ is the particular version:
 
 ### Alerts
 
-Network alerts should be placed in `_alerts/YYYY-MM-DD-SHORTITLE.html` and adhere to this format:
+1. [Who to contact](#who-to-contact)
+2. [Basic alert](#basic-alert) (emergency fast instructions)
+3. [Detailed alert](#detailed-alert)
+4. [Clearing an alert](#clearing-an-alert)
 
-```
+#### Who to Contact
+
+The following people can publish alerts on Bitcoin.org.  Their email
+addresses are on the linked GitHub profiles.
+
+- Saïvann Carignan, [@saivann](https://github.com/saivann), saivann on Freenode
+- Dave Harding, [@harding](https://github.com/harding), harding on Freenode
+- Wladimir van der Laan, [@laanwj](https://github.com/laanwj), wumpus on Freenode
+- Theymos, [@theymos](https://github.com/theymos), theymos on Freenode
```

Comment:
I noticed @theymos's email address isn't on his GitHub profile as suggested above. @theymos, is it possible for you to add the information, just in case?